### DISCUSSION TITLE
[misc] Which file system is preferable for volume storage ? (xfs,btrfs,ext4,ext3,....)

### DISCUSSION THREAD

**Discussion by ldapomni** [seaweedfs/seaweedfs#2121] [Q&A]

**Comment by ijunaid8989** [comment]
we have been using `xfs` with RAID6, for the past 5 years and it was pretty good same for filer as well as for storage.

we have now moved to zfs raidZ2. and it's going pretty good as well.

FYI: we also got into a crash with `xfs` in past, but only twice in 5 years, which was recoverable, we lost a few volume files though but we didn't lose the data to some extent.
